test/fixtures/rules/std_rule_cases/L068.yml

```diff
@@ -0,0 +1,82 @@
+rule: L068
+
+test_pass_one_line_subquery:
+  pass_str: |
+      SELECT A.SOMETHING
+      FROM (SELECT B FROM D GROUP BY 1) AS A
+
+# The fix of indentation rely on L003
+# If there is a subquery inside the bracket, the fix also rely on L067
+test_fail_example_in_description:
```

Comment:
Can you add tests where there are comments around the area the rule considers? Handling comments is often tricky. The rule may either:
* Handle comments correctly (harder)
* Detect comments and not try to auto-fix (easier)